lib/browser/api/app.js

```diff
@@ -25,6 +25,7 @@ Object.assign(app, {
     return Menu.getApplicationMenu()
   },
   commandLine: {
+    ...process.atomBinding('command_line'),
     appendSwitch (...args) {
```

Comment:
You could move these methods implemented via `app` module into `command_line` module in a followup PR.

Reply:
done in this PR